tools/dockerfile/grpc_scan_build/Dockerfile

```diff
@@ -41,6 +41,19 @@ RUN git clone --recursive https://github.com/grpc/grpc.git
 EXPOSE 8181
 
 CMD \
+  # creating a repository for gRPC submodules for local cloning
+  # this list should be updated when the list of submodules for gRPC changes
+  mkdir grpc-submodules && cd grpc-submodules && \
```

Comment:
Another problem is you are manually listing all the submodules here. The day we add a new submodule, there's a very high chance we will forget to manually update the list.